CONTRIBUTING.md

```diff
@@ -1,92 +1,77 @@
 # Contributing
 
-- [How to Contribute](#how-to-contribute)
-  * [Getting Code](#getting-code)
-  * [Code reviews](#code-reviews)
-  * [Code Style](#code-style)
-  * [API guidelines](#api-guidelines)
-  * [Commit Messages](#commit-messages)
-  * [Writing Documentation](#writing-documentation)
-  * [Adding New Dependencies](#adding-new-dependencies)
-  * [Running & Writing Tests](#running--writing-tests)
-  * [Public API Coverage](#public-api-coverage)
-- [Contributor License Agreement](#contributor-license-agreement)
-  * [Code of Conduct](#code-of-conduct)
+## Choose an issue
 
-## How to Contribute
+Playwright **requires an issue** for every contribution, except for minor documentation updates. We strongly recommend to pick an issue labeled `open-to-a-pull-request` for your first contribution to the project.
 
-We strongly recommend that you open an issue before beginning any code modifications. This is particularly important if the changes involve complex logic or if the existing code isn't immediately clear. By doing so, we can discuss and agree upon the best approach to address a bug or implement a feature, ensuring that our efforts are aligned.
+If you are passioned about a bug/feature, but cannot find an issue describing it, **file an issue first**. This will facilitate the discussion and you might get some early feedback from project maintainers before spending your time on creating a pull request.
 
-### Getting Code
-
-Make sure you're running Node.js 20 to verify and upgrade NPM do:
+## Make a change
 
+Make sure you're running Node.js 20 or later.
 ```bash
 node --version
-npm --version
-npm i -g npm@latest
 ```
 
-1. Clone this repository
-
-    ```bash
-    git clone https://github.com/microsoft/playwright
-    cd playwright
-    ```
+Clone the repository. If you plan to send a pull request, it might be better to [fork the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) first.
```

Comment:
Usually you clone our repository and then add your fork as an origin / VSCode will automatically fork the repository for you if try to push to microsoft. I'd not mention forking here at all.